scripts/ft-benchmark.sh

```diff
@@ -32,7 +32,7 @@ make neard
 
 # Start neard
 nearup run localnet --binary-path target/release/ --num-nodes 1 --num-shards 1 --override
-
+python3 scripts/ft-bechmark-data-sender.py
```

Comment:
Hm, shouldn't this run after the Locust launch? In my understanding, right now this will run right after the `nearup run` and will report 0 TPS to the DB.

It's fine if this is temporary, but I think it's worth discussing how we intend to run this in a final version.

Reply:
On my experience locust start + reaching target amount of users takes +- 5 min. If, let say, we have 400 tps, and experiment for 2 hours, and tps increasing linearly while amount of users grow, this script will show `(400 * (60+55) + 200 * 5) / 120 = 391.6`, which is probably reasonable difference.

I think in this case it is even better to run data collector ~5 min after locust to ignore time when locust is rumping up